src/services/GameService.js

```diff
@@ -0,0 +1,77 @@
+const DEFAULT_HEADER_JSON = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json',
+};
+
+/**
+ * @class GameService
+ */
+export default class GameService {
```

Comment:
Why use a class here ?

You could export distinct functions factories, something like:

```js
export default newGame = (baseUrl = defaultBaseUrl) => (mode = 'multi') => { ... };
```